### PR TITLE
Updates for Linkerd

### PR DESCRIPTION
**Pull request by kflynn**
Updates after releasing Linkerd 2.13

Signed-off-by: Flynn <flynn@buoyant.io>

### CODE REVIEW

File: _includes/comparison.html [hunk 1]
```diff
@@ -226,7 +226,7 @@ <h2 class="section-heading--primary"> Service Mesh Comparison </h2>
       <tr>
         <th scope="row">Cloud Integrations</th>
         <td><a href="https://cloud.google.com/istio" target="_blank">Google Cloud</a>, <a href="https://www.alibabacloud.com/help/doc-detail/89805.htm" target="_blank">Alibaba Cloud</a>, <a href="https://cloud.ibm.com/docs/containers?topic=containers-istio" target="_blank">IBM Cloud</a></td>
-        <td><a href="https://marketplace.digitalocean.com/apps/linkerd" target="_blank">DigitalOcean</a></td>
+        <td><a href="https://marketplace.digitalocean.com/apps/linkerd-beta" target="_blank">DigitalOcean</a></td>
```
Comment (MarcoSteinke) [inline _includes/comparison.html:229]: Hello,
can you tell me if there is more meaning behind this change in the given URL to `linkerd-beta` ? I would like to know more about this, but can not find information.

You can contact me via mail (marco.steinke@innoq.com)

File: _includes/comparison.html [hunk 2]
```diff
@@ -445,7 +445,7 @@ <h2 class="section-heading--primary"> Service Mesh Comparison </h2>
       <tr>
         <th scope="row">Header- and Path-based Traffic Splits<small class="form-text">Routing rules based on request header and path</small></th>
         <td><a href="https://istio.io/docs/tasks/traffic-management/request-routing/" target="_blank">yes</a></td>
-        <td>planned</td>
+        <td><a href="https://linkerd.io/2.13/tasks/configuring-dynamic-request-routing/" target="_blank">yes</a>, through <a href="https://gateway-api.sigs.k8s.io/guides/http-routing/" _target="blank">Gateway API</a></td>
```
Comment (MarcoSteinke) [inline _includes/comparison.html:448]: 👍🏻

**Review by MarcoSteinke — APPROVED** [review]
Thank you for the updates of the information on Linkerd 👍🏻

File: _includes/comparison.html (hunk 1 above)
Comment (kflynn) [reply _includes/comparison.html:229]: When I looked at the Digital Ocean page, the `linkerd-beta` page works, and without the `-beta` it's a broken link. 🤷‍♂️

**Pull request merged**